src/app/models/usermodel.js

```diff
@@ -25,46 +25,46 @@ function cached(fn) {
-        //     full_name: 'Roy Johnson',
-        //     uuid: '53cdf5b7-6dd9-45a5-bad7-e1f5532180e2',
-        //     support_identifier: 'cs_5bcab9be',
-        //     is_not_gdpr_location: true,
-        //     opt_out_of_cookies: false,
-        //     using_openstax: false,
-        //     salesforce_contact_id: '0037h00000SEXNqAAP',
-        //     // May be confirmed_faculty, rejected_by_sheerid, incomplete_signup
-        //     faculty_status: 'incomplete_signup',
-        //     is_newflow: true,
-        //     is_instructor_verification_stale: false,
-        //     needs_complete_edu_profile: false,
-        //     self_reported_role: 'instructor',
-        //     self_reported_school: '',
-        //     school_type: 'college',
-        //     school_location: 'domestic_school',
-        //     is_kip: false,
-        //     is_administrator: false,
-        //     grant_tutor_access: false,
-        //     contact_infos: [
-        //         {
-        //             id: 61694,
-        //             type: 'EmailAddress',
-        //             value: 'rej2+mos1@rice.edu',
-        //             is_verified: true,
-        //             is_guessed_preferred: true
-        //         }
-        //     ],
-        //     applications: [
-        //         {
-        //             id: 5,
-        //             name: 'OpenStax CMS Dev'
-        //         }
-        //     ]
-        // });
+        return Promise.resolve({
+            id: 10060116,
+            name: 'Roy Johnson',
+            first_name: 'Roy',
+            last_name: 'Johnson',
+            full_name: 'Roy Johnson',
+            uuid: '53cdf5b7-6dd9-45a5-bad7-e1f5532180e2',
+            support_identifier: 'cs_5bcab9be',
+            is_not_gdpr_location: true,
+            opt_out_of_cookies: false,
+            using_openstax: false,
+            salesforce_contact_id: '0037h00000SEXNqAAP',
+            // May be confirmed_faculty, rejected_by_sheerid, incomplete_signup
+            faculty_status: 'confirmed_faculty',
+            is_newflow: true,
+            is_instructor_verification_stale: false,
+            needs_complete_edu_profile: false,
+            self_reported_role: 'instructor',
+            self_reported_school: '',
+            school_type: 'college',
+            school_location: 'domestic_school',
+            is_kip: false,
+            is_administrator: false,
+            grant_tutor_access: false,
+            contact_infos: [
+                {
+                    id: 61694,
+                    type: 'EmailAddress',
+                    value: 'rej2+mos1@rice.edu',
+                    is_verified: true,
+                    is_guessed_preferred: true
+                }
+            ],
+            applications: [
+                {
+                    id: 5,
+                    name: 'OpenStax CMS Dev'
+                }
+            ]
+        });
         // eslint-disable-next-line no-unreachable
```

Comment:
heh, I'm guessing this was left enabled by mistake 😁

Reply:
I knew there would come a day that I would wind up committing that. Thanks.